x-pack/platform/plugins/shared/onechat/public/embeddable/embeddable_conversation.tsx

```diff
@@ -17,37 +17,50 @@ export const EmbeddableConversationInternal: React.FC<EmbeddableConversationInte
   props
 ) => {
   const { euiTheme } = useEuiTheme();
+  const { onClose, ariaLabelledBy } = props;
 
   const backgroundStyles = css`
     background-color: ${euiTheme.colors.backgroundBasePlain};
   `;
 
-  const headerHeight = `calc(${euiTheme.size.xl} * 2)`;
   const headerStyles = css`
     ${backgroundStyles}
     display: flex;
     align-items: center;
-    block-size: ${headerHeight};
-    padding: 0 ${euiTheme.size.base};
+    &.euiFlyoutHeader {
+      padding-inline: 0;
+      padding-block-start: 0;
+      padding: ${euiTheme.size.base};
+    }
   `;
-  const contentStyles = css`
+  const bodyStyles = css`
     ${backgroundStyles}
-    width: 100%;
-    height: 100%;
-    max-block-size: calc(var(--kbn-application--content-height) - ${headerHeight});
-    display: flex;
-    justify-content: center;
-    align-items: center;
+    flex: 1;
+
+    .euiFlyoutBody__overflow {
+      overflow: hidden;
+      height: 100%;
+    }
+
+    .euiFlyoutBody__overflowContent {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      height: 100%;
+      overflow: hidden;
+      padding: 0;
+    }
```

Comment:
The flyout body tries to handle overflow and scrolling etc., internally, and this isn't something we can have because the `Conversation.tsx` component is responsible for it's own scrolling, that's why these overrides were ncessary.  Once we separate the ConversationInputForm from the resizable container, maybe things will be easier.